tests/tests.schema

```diff
@@ -9180,6 +9183,9 @@
         },
         {
           "$ref": "#/definitions/Microsoft.Test.UserSays"
+        },
+                {
+          "$ref": "#/definitions/Microsoft.Test.UserEvent"
         },
         {
```

Comment:
I suspect you did this by hand.  You should not.  What you should do is install the latest bf cli and run the script which creates this file.